package.json

```diff
@@ -1,27 +1,27 @@
 {
         "name": "mako",
         "description": "Storage backend for Manta",
-        "version": "1.2.0",
+        "version": "2.0.0",
         "author": "Joyent (joyent.com)",
         "license": "MPL-2.0",
         "private": true,
         "dependencies": {
                 "assert-plus": "1.0.0",
                 "bunyan": "1.8.12",
-                "carrier": "0.1.7",
-                "manta": "5.2.1",
-                "posix-getopt": "1.0.0",
+                "kthxbai": "0.4.0",
                 "restify": "4.3.4",
+                "@smaller/tap": "11.1.4-1.0.0",
                 "triton-metrics": "1.0.2",
-                "vasync": "2.2.0"
+                "uuid": "3.3.3",
+                "vasync": "2.2.0",
+                "verror": "1.10.0"
         },
         "devDependencies": {
-                "async": "0.1.x",
-                "tap": "0.4.0",
-                "tape": "4.6.3",
-                "node-uuid": "1.4.0",
-                "extsprintf": "1.3.0",
-                "jsprim": "1.3.1"
+                "eslint": "^4.19.1",
+                "eslint-config-prettier": "^6.0.0",
+                "eslint-plugin-joyent": "^2.1.0",
+                "eslint-plugin-prettier": "^3.1.0",
+                "prettier": "^1.18.2"
         },
         "sdcDependencies": {
                 "config-agent": ">=1.2.0"
```

Comment:
Update "engines" block below... or just drop it.

Reply:
sure.